README.md

```diff
@@ -6,9 +6,9 @@ Simplify
 Generic Android Deobfuscator
 ----------------------------
 
-Simplify uses a virtual machine to understand what an app does. Then, it applies optimizations to create code that behaves identically, but is easier for a human to understand. Specifically, it takes Smali files as input and outputs a Dex file with (hopefully) identical semantics but less complicated structure.
+**Simplify** is a framework that utilizes a virtual machine to understand what an app does and then applies optimizations to create code that is easier for a human to understand while maintaining the identical behavior of the application. It takes Smali files as the input and outputs a Dex file with (hopefully) identical semantics but a less complicated/obfuscated structure.
 
-For example, if an app's strings are encrypted, Simplify will interpret the app in its own virtual machine to determine semantics. Then, it uses the apps own code to decrypt the strings and replaces the encrypted strings and the decryption method calls with the decrypted versions. It's a generic deobfuscator because Simplify doesn't need to know how the decryption works ahead of time. This technique also works well for eliminating different types of white noise, such as no-ops and useless arithmetic.
+For example, if an app's strings are encrypted, Simplify will interpret the app in its own virtual machine to determine the semantics. Then, it will use the app's own decryption code to decrypt the strings and replace them along with their decryption method invocations with the decrypted equivalents. Simplify is a **generic** deobfuscator because it doesn't need to know how the decryption works ahead of time. The underlying technique also works well for eliminating different types of white noise such as no-ops and useless arithmetics.
```

Comment:
Oooo, some grammar fixes in there. I like emphasizing "generic".

The second sentence switches out of the hypothetical tense and into a concrete tense for directness.

"own decryption code to decrypt" seems redundant, but it adds clarity. I like it.

I favor "calls" rather than "invocations" because it's simpler. The word "decrypt" shows up 4 times in this sentence.

"The underlying technique" adds words and indirection without adding anything

arithmetics should be singular. I think "maths" is a strange, britishism? I've never heard "arithmetics".